src/Navbar/Navbar.js

```diff
@@ -1,12 +1,13 @@
 import './Navbar.css'
 import SearchByNameForm from '../SearchByTitleForm/SearchByTitleForm';
+import { NavLink } from 'react-router-dom';
 
-const Navbar = ({ view, goHome, findMovieByTitle }) => {
+const Navbar = ({ findMovieByTitle }) => {
     return (
         <nav className='navbar'>
             <h1>Rancid Tomatillos</h1>
-            {view && <button className='navbar-home' onClick={goHome}>Home</button>}
-            {view || <SearchByNameForm findMovieByTitle={findMovieByTitle} />}
+            <NavLink to='/' className='navbar-home'>Home</NavLink>
+            <SearchByNameForm findMovieByTitle={findMovieByTitle} />
```

Comment:
I am not sure how to make the Home button only work on the Movie page and the Search button work on the All Movie page. We may be able to make them both make sense on work on both pages. Maybe just disable the home button on the home page?

Reply:
Yeah do you think maybe an inline style on it on the home page to make it invisible?

Reply:
This is the issue you solved that you msged me about last night, right?